payment-method-messaging/src/main/java/com/stripe/android/paymentmethodmessaging/view/PaymentMethodMessagingViewModel.kt

```diff
@@ -0,0 +1,63 @@
+package com.stripe.android.paymentmethodmessaging.view
+
+import android.app.Application
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewmodel.CreationExtras
+import com.stripe.android.core.exception.StripeException
+import com.stripe.android.core.networking.ApiRequest
+import com.stripe.android.model.PaymentMethodMessage
+import com.stripe.android.networking.StripeApiRepository
+import com.stripe.android.paymentmethodmessaging.view.injection.DaggerPaymentMethodMessagingComponent
+import com.stripe.android.ui.core.isSystemDarkTheme
+import com.stripe.android.utils.requireApplication
+import javax.inject.Inject
+
+internal class PaymentMethodMessagingViewModel @Inject constructor(
+    private val application: Application,
```

Comment:
Nit: Can we inject a provider method that returns whether to use the dark theme instead?